src/components/FeaturePanel/ImageSection/Photo.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { Image } from '../../../services/types';
+
+interface PhotoInnerProps {
+  image: Image;
+}
+
+const InnerPano: React.FC<PhotoInnerProps> = ({ image }) => {
+  const [url, setUrl] = React.useState('');
+  const [pannellumUrl, setPannellumUrl] = React.useState<string | null>(null);
+
+  React.useEffect(() => {
+    setUrl(`${window.location.protocol}//${window.location.host}`);
+  }, []);
+
+  React.useEffect(() => {
+    setPannellumUrl(
+      `https://cdn.pannellum.org/2.5/pannellum.htm#panorama=${encodeURIComponent(
+        image.thumb,
+      )}&config=${encodeURIComponent(`${url}/pannellum-config.json`)}`,
+    );
+  }, [url]);
+
+  return (
+    <>
+      {pannellumUrl && (
+        <iframe
```

Comment:
Also when I open this link https://osmapp-git-fork-dlurak-panorama-osm-app-team.vercel.app/node/6504183373 - i get the following bug. Do you have any idea why could this happen? 

It looks like it happens only in SSR mode. But useEffect is only executed once on the client. 🤔  .. the above suggestion remove url state may help with this.

![image](https://github.com/zbycz/osmapp/assets/385047/58f5f5fd-ebb2-4110-91e5-c1c4923815f1)

Reply:
Very very weird, I can't even reproduce it :/
When I open the very same link it works perfectly fine

Reply:
It works for me now too. Could it be perhaps by mapillary returning an empty set? I remember it happend from time to time.

Nevertheless, we can let it be and observe if it ever happens again :)